src/partition/ReceivedPartition.cpp

```diff
@@ -89,11 +89,20 @@ void ReceivedPartition::compute()
 {
   PRECICE_TRACE();
 
+  std::cout << "This partition is provided" << std::endl;
```

Comment:
Mmh. I would still call such a partition (or better mesh) received and not provided. What is provided is only the "domain of interest" (which one could call "partition" again, I agree, but this would lead to many misunderstandings I guess).

Reply:
I guess this changes intuitively when omitting the mapping